### ISSUE TITLE
Cross-Origin Request Blocked for exclusion-orders.geojson

### ISSUE THREAD

**Issue by gjost** [denshoproject/sites-of-shame#128]
TL;DR I'm getting CORS errors for `exclusion-orders.geojson` from our API subdomain.

I've been setting up the React app on our servers in preparation for launch.  We have the https://github.com/stamen/densho-sites-of-shame repository checked out on a back-of-house server with Yarn installed.  We do the build there and rsync it to our static files (Nginx) server.  We have another Nginx server out in front to direct requests to our various application servers.

The app works as delivered.  However we have a requirement that the app data should reside on a separate subdomain, namely https://api.densho.org/sitesofshame/.

I modified `.env.production` thusly...
```
    REACT_APP_MAPBOX_ACCESS_TOKEN=REDACTED
    REACT_APP_MAPBOX_BASE_LAYER=REDACTED
    REACT_APP_DATA_PATH=https://api.densho.org/sitesofshame/
    REACT_APP_ROUTER_MODE=hash
    PUBLIC_URL=./
```
...ran `yarn build:production`, and rsynced the `build/` dir to where it needs to go.

The front-of-house Nginx server configs are modified to include the CORS headers.
```
    upstream ddr_media {
        server REDACTED_IP:80 fail_timeout=0;
    }

    server {
        listen      443 ssl;
        server_name api.densho.org api.local;
        ...
        add_header  X-Request-ID $request_id;
    
        location / {
            if ($request_method = 'OPTIONS') {
                add_header Access-Control-Allow-Origin *;
                add_header Access-Control-Allow-Methods "GET, POST, OPTIONS";
                add_header Access-Control-Allow-Headers "DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range";
                add_header Access-Control-Max-Age 1728000;
                add_header Content-Type "text/plain; charset=utf-8";
                add_header Content-Length 0;
                return 204;
            }
            if ($request_method = 'POST') {
                add_header Access-Control-Allow-Origin *;
                add_header Access-Control-Allow-Methods "GET, POST, OPTIONS";
                add_header Access-Control-Allow-Headers "DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range";
                add_header Access-Control-Expose-Headers "Content-Length,Content-Range";
            }
            if ($request_method = 'GET') {
                add_header Access-Control-Allow-Origin *;
                add_header Access-Control-Allow-Methods "GET, POST, OPTIONS";
                add_header Access-Control-Allow-Headers "DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range";
                add_header Access-Control-Expose-Headers "Content-Length,Content-Range";
            }
            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;

            proxy_set_header Host $http_host;
            proxy_redirect off;
            proxy_pass http://ddr_media;
            break;
        }
    }
```
The static media server's Nginx does not add or modify any headers.

PROBLEM: I'm getting the following error in Firefox:
> Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://api.densho.org/sitesofshame/exclusion-orders.geojson. (Reason: CORS header ‘Access-Control-Allow-Origin’ does not match ‘*, *’).

This error only appears to affect `exclusion-orders.geojson` -- other files from the API seem to work fine.

We've done one project earlier that involved CORS headers but we handled that using a Django app.  Obviously that's not an option here.

Any thoughts? Advice?  Thanks in advance for your help!
